test/client-test.ts

```diff
@@ -251,7 +251,7 @@ describe('RSP Client', () => {
 
             const result = await client.createServerSync('path', 'id');
 
-            expect(modelStub.createServerFromPath).calledWith('path', 'id', defaultTimeout);
+            expect(modelStub.createServerFromPath).calledWith('path', 'id', undefined, defaultTimeout);
```

Comment:
These methods should be tested with some attributes as well, so we can see if they are passed correctly. Either rewrite these test cases to use attributes, or create separate. I don't mind either :)